administrator-manual/en/suricata.rst

```diff
@@ -233,7 +233,20 @@ Web Specific Apps
 WORM
   Traffic indicative of network based worm activity.
 
+Bypass
+======
+
+The bypass disables IPS protection for selected hosts: all traffic from/to the given host will not be analyzed.
+
+To create a bypass access the new Server Manager and open the :guilabel:`IPS` application, then go to the :guilabel:`Bypass`
```

Comment:
```suggestion
To create a bypass access the Server Manager and open the :guilabel:`IPS` application, then go to the :guilabel:`Bypass`
```

Reply:
Too late for this. If you want such change, please open a PR on master branch.